### PR TITLE
Fixes bug 1225567 - Added a note that the product summary does not us…

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
…e filters.

@peterbe r?

### CODE REVIEW

File: webapp-django/crashstats/signature/templates/signature/signature_summary.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
 <div class="panel">
     <header class="title">
-        <h2>Product</h2>
+        <h2>Product *</h2>
```
Comment (peterbe) [inline webapp-django/crashstats/signature/templates/signature/signature_summary.html]: You should use `<h2>Product<sup>*</sup></h2>` so you get a raised character. 
And use that too at the beginning of the text below.

File: webapp-django/crashstats/signature/templates/signature/signature_summary.html [hunk 2]
```diff
@@ -77,6 +77,9 @@
                 {% endfor %}
             </tbody>
         </table>
+        <em>
```
Comment (peterbe) [inline webapp-django/crashstats/signature/templates/signature/signature_summary.html]: `em` is not a blocking tag. By the way, `em` (and `strong`) is a relic from the XHTML days. The more modern one to use is `i` (and `b`).

Comment (peterbe) [inline webapp-django/crashstats/signature/templates/signature/signature_summary.html]: I.e. I think it should be: `<p class="footnote">` and something like `p.footnote { font-style: italic; }`

**Comment by AdrianGaudebert** [comment]
@peterbe updated!

**Comment by peterbe** [comment]
r+ but before you merge, consider changing the word "week" to 7 days. 
Because the summary isn't from Monday to today. It's from (today - 7days)  to today.

**Pull request merged**